### DISCUSSION TITLE
Self-host installation not allowing any ctions

### DISCUSSION THREAD

**Discussion by ash-development** [Budibase/budibase#14332] [Help]
**Checklist**
- [x] I have searched budibase discussions and github issues to check if my issue already exists

**Hosting**

- Self
  - Method: <method>docker compose
  - Budibase Version: <version> 2.29.29 

**Describe the bug**
When trying to delete an app, i get these errors:
```[Error] Failed to load resource: the server responded with a status of 404 () (app_dev_e040fda5f8d6491081bf987760c78cb3, line 0)
[Warning] [Builder] HTTP 404 on DELETE:https://forms.ashie.lol/api/applications/app_dev_e040fda5f8d6491081bf987760c78cb3 (index-09ae2f0b.js, line 1)
	The specified bucket does not exist
```

When trying to access said apps, I get these errors
```
[Error] Failed to load resource: the server responded with a status of 404 () (appPackage, line 0)
[Error] Failed to load resource: the server responded with a status of 404 () (sync, line 0)
[Warning] [Builder] HTTP 404 on GET:https://forms.ashie.lol/api/applications/app_dev_e040fda5f8d6491081bf987760c78cb3/appPackage (index-09ae2f0b.js, line 1)
	CouchDB error: missing
[Warning] [Builder] HTTP 404 on POST:https://forms.ashie.lol/api/applications/app_dev_e040fda5f8d6491081bf987760c78cb3/sync (index-09ae2f0b.js, line 1)
	CouchDB error: missing
[Error] Failed to load resource: the server responded with a status of 404 () (appPackage, line 0)
[Warning] [Builder] HTTP 404 on GET:https://forms.ashie.lol/api/applications/app_dev_e040fda5f8d6491081bf987760c78cb3/appPackage (index-09ae2f0b.js, line 1)
	CouchDB error: missing
```

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] Mac
 - Browser [e.g. chrome, safari] Safari
 - Version [e.g. 22] 17.3

**Additional context**
My installation is brand new and I'm using Caddy as a reverse proxy

**Comment by linear** [comment]
<p><a href="https://linear.app/budibase/issue/BUDI-8542/self-host-installation-not-allowing-any-ctions">BUDI-8542 Self-host installation not allowing any ctions</a></p>
